### ISSUE TITLE
[False positive] Missing ABAP Doc for method calculate

### ISSUE THREAD

**Issue by lucasborin** [abaplint/abaplint#2732]
Given the following interface:
```abap
REPORT zfoo.

"! <p class="shorttext synchronized" lang="en">Text</p>
INTERFACE
  if_test PUBLIC.

  "! <p class="shorttext synchronized" lang="en">Text</p>
  METHODS
    calculate
      RAISING   cx_test.

ENDINTERFACE.
```

When I run abaplint, it reports:
```
/zfoobar.prog.abap[9, 5]: Missing ABAP Doc for method calculate(abapdoc)
```

I would expect to not find any issue, as it has a single method. It would make sense if I had more than one method in the interface.

**Comment by larshp** [comment]
@lucasborin try again, 2.93.45

**Comment by lucasborin** [comment]
Working fine, thanks Lars!
